src/constants.ts

```diff
@@ -1,5 +1,127 @@
+import path from 'path';
+
+/**
+ * The name of this project. Used to exclude this repo as a lintable project.
+ */
+const THIS_PROJECT_NAME = 'module-lint';
+
+/**
+ * The root directory of this project.
+ */
+const THIS_PROJECT_DIRECTORY_PATH = path.resolve(__dirname, '..');
+
 /**
  * The number of milliseconds in an hour, used to determine when to pull the
  * latest changes for previously cached repositories.
  */
 export const ONE_HOUR = 60 * 60 * 1000;
+
+/**
+ * Wherever the tool was run.
+ */
+export const WORKING_DIRECTORY_PATH = process.cwd();
+
+/**
+ * The usage text printed when the user requests help or provides invalid input.
+ */
+export const USAGE_TEXT = `
+Analyzes one or more repos for divergence from a template repo.
+
+${THIS_PROJECT_NAME} OPTIONS [ARGUMENTS...]
+
+Pass the names of one or more MetaMask repositories to lint them, or pass
+nothing to lint all MetaMask repositories.
+`.trim();
+
+/**
+ * In order to lint a remote repository, that repository must be cloned first.
+ * This is the temporary directory where the clone lives.
+ */
+export const DEFAULT_CACHED_REPOSITORIES_DIRECTORY_PATH = path.join(
+  THIS_PROJECT_DIRECTORY_PATH,
+  'tmp/repositories',
```

Comment:
For platform cross-compatibility, we shouldn't embed a delimiter here. Something like this should be equivalent except with better platform support:

```suggestion
  'tmp', 'repositories',
```

Reply:
Not sure how I ended up looking at this PR, but `path.join` normalizes paths to the path separator for the OS.

Reply:
Ah, that's good. Well, it doesn't hurt, so I'll accept this suggestion.